app/car.py

```diff
@@ -0,0 +1,14 @@
+import math
+
+
+class Car:
+    def __init__(self, brand: str, fuel_consumption: float) -> None:
+        self.brand = brand
+        self.fuel_consumption = fuel_consumption
+
+    def fuel_cost(self, distance: float, fuel_price: float) -> float:
+        return (distance / 100) * fuel_price * self.fuel_consumption
+
+    @staticmethod
+    def calculate_distance(home: list, shop: list) -> float:
+        return math.sqrt((shop[0] - home[0]) ** 2 + (shop[1] - home[1]) ** 2)

```

Comment:
The `calculate_distance` method assumes that both `home` and `shop` lists have exactly two elements (representing x and y coordinates). It would be safer to validate the length of these lists before performing calculations to avoid potential index errors.